schema/data/prod/product.sql

```diff
@@ -39,7 +39,11 @@ values
 (33, 'Petroleum Refining','BC refining complexity factor unit','active',true),
 (34, 'Wood pellets', 'bone dry tonnes','active', true),
 (35, 'Wood chips', 'bone dry tonnes','active', true),
-(36, 'Waste rendering', 'tonnes','active', true)
+(36, 'Waste rendering', 'tonnes','active', true),
+(37, 'Imported electricity', 'MWh', 'active' ,true),
+(38, 'Imported heat', 'GJ', 'active' ,true),
+(39, 'Generated electricity', 'MWh', 'active' ,true),
+(40, 'Generated heat', 'GJ', 'active' ,true)
```

Comment:
Add exported heat/electricity

Reply:
exported heat/electricity were already entered as products in the data/prod/products.sql file (id's 10 & 11)

Reply:
Right, I didn't remember if I had done that or not